dnf-behave-tests/dnf/do-installed-from-repo.feature

```diff
@@ -0,0 +1,196 @@
+Feature: "do" command with "--installed-from-repo" and "--from-repo"
+
+
+Background: Install some RPMs from repositories
+  Given I use repository "from-repo"
+   When I execute dnf with args "do --action=install abcde pkgB"
+   Then the exit code is 0
+    And Transaction is following
+        | Action        | Package                               |
+        | install       | abcde-0:2.9.2-1.fc29.noarch           |
+        | install       | pkgB-0:1.0-1.fc29.x86_64              |
+        | install-dep   | dependency1-0:1.0-1.fc29.x86_64       |
+        | install-dep   | wget-0:1.19.5-5.fc29.x86_64           |
+        | install-weak  | flac-0:1.3.2-8.fc29.x86_64            |
+  Given I use repository "from-repo-updates1"
+    And I use repository "from-repo-updates2"
+   When I execute dnf with args "do --action=install basesystem"
+   Then the exit code is 0
+    And Transaction is following
+        | Action        | Package                               |
+        | install       | basesystem-0:12-2.fc29.noarch         |
+        | install-dep   | filesystem-0:3.10-1.fc29.x86_64       |
+        | install-dep   | setup-0:2.14.2-1.fc29.noarch          |
+
+
+Scenario: Upgrade pkgB installed from "from-repo-updates1" and install new dependency
```

Comment:
The scenario description doesn't match - should be "from-repo" instead of "from-repo-updates1".

Reply:
Fixed. Thaks